actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java

```diff
@@ -0,0 +1,83 @@
+package org.sunbird.user.actors;
+
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.*;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class helps in interacting with adding and validating the user-certificate details
+ */
+@ActorConfig(
+        tasks = {"validateCertificate"},
+        asyncTasks = {}
+)
+public class CertificateActor extends UserBaseActor{
+
+  private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+  @Override
+  public void onReceive(Request request) throws Throwable {
+    if(request.getOperation().equals(ActorOperations.VALIDATE_CERTIFICATE.getValue())) {
+      getCertificate(request);
+    }
+
+  }
+
+  /** This method validates the access-code of the certificate and retrieve certificate details.
+   * @param userRequest
+   */
+  private void getCertificate(Request userRequest) {
+    Map request = userRequest.getRequest();
+    Map userResponse = new HashMap<String, Object>();
```

Comment:
Map userResponse = new HashMap<String, Object>(); 
and 
Response userResult = new Response();
can be define inside if block

Reply:
Alright. Done.